README.md

```diff
@@ -7,8 +7,6 @@ Fluxxed_up [![Build Status](https://travis-ci.org/everplans/fluxxed_up.svg?branc
 - [What's Included](#whats-included)
   - [Action Prototype](#actionprototype)
   - [Dispatcher](#dispatcher)
-  - [ExtraStorage](#extrastorage)
```

Comment:
Neither of these were being used in this repo and we don't bring them into web-apps ever so I'm just removing them. They are just dead code at this point so there's no reason to keep them in the bundle.